shaders.go

```diff
@@ -122,7 +119,12 @@ void main (void) {
 }
 
 func (s *basicShader) Pre() {
+	// Enable shader and buffer, enable attributes in shader
 	Gl.UseProgram(s.program)
+	Gl.BindBuffer(Gl.ELEMENT_ARRAY_BUFFER, s.indexVBO)
+	Gl.EnableVertexAttribArray(s.inPosition)
+	Gl.EnableVertexAttribArray(s.inTexCoords)
+	Gl.EnableVertexAttribArray(s.inColor)
```

Comment:
Best-practice is to `Enable` and `Disable` those whenever we switch shaders. If we don't, other shaders won't run without throwing errors.